monitor/core/taps/bgpstreamhist.py

```diff
@@ -148,6 +163,7 @@ def parse_bgpstreamhist_csvs(self):
                                                         msg
                                                     )
                                                 )
+                                            break
```

Comment:
@slowr at lalst found the reason why we received duplicate updates from BGPstream, it was because we did not break in the for-loop for the prefixes! (one of the reasons is this :) )